### ISSUE TITLE
No compatible binary build found for this package

### ISSUE THREAD

**Issue by jorisroling** [nicolaslopezj/roles#13]
while doing `meteor update`

```
While checking for nicolaslopezj:roles@2.1.0:
error: No compatible binary build found for this package. Contact the package author and ask them to publish it for your platform.
```

**Comment by PersonaA** [comment]
I have this too.

**Comment by ashishapy** [comment]
same here.
error when I do `meteor update`

`=> Errors while upgrading packages:           
While checking for nicolaslopezj:roles@2.1.0:
error: No compatible binary build found for this package. Contact the package author and ask them to publish it for your platform.`

**Comment by nicolaslopezj** [comment]
Fixed with 2.1.1
